web/src/display-defaults/roadmap/business-formation/BusinessFormationDefaults.ts

```diff
@@ -1,12 +1,12 @@
 export const BusinessFormationDefaults: Record<string, string> = {
+  llcText: "LLC",
```

Comment:
Design requires 'LLC' to be displayed instead of the name within the legal structure object - "Limited Liability Company (LLC)". 

I decided to add the label within the defaults file. Is this reasonable? I could see us using LLC in the future, along with LLP and LP in other parts of the app. Does it make sense to update the type to include a shorthandName key? 

https://www.figma.com/file/408Krq8roicOzFVYrjwIdE/Navigator-Designs?node-id=7156%3A192250

Reply:
@aloverso do you have any thoughts on this?

Reply:
what you have here seems reasonable for now, but probably when we implement the next one, I like the idea of using a "shorthand name" property instead